README.md

```diff
@@ -22,10 +22,8 @@ If you want to build the repository from source follow these instructions:
     # Ensure NodeJS is downloaded: https://nodejs.org/en/download/
     git clone https://github.com/brownhci/WebGazer.git
     cd WebGazer
-    #install the dependencies
+    #install the dependencies and triggers building the project
```

Comment:
I actually don't think this is the case for the FIRST install, but I'll need to check. So we should leave the part about num run build

Reply:
You're right, cause husky isn't set up. Yeah let's leave it then.

Btw do you know what's up with these tensorflow warnings?

npm WARN @tensorflow-models/blazeface@0.0.5 requires a peer of @tensorflow/tfjs-core@^1.5.2 but none is installed. You must install peer dependencies yourself.
npm WARN @tensorflow-models/blazeface@0.0.5 requires a peer of @tensorflow/tfjs-converter@^1.5.2 but none is installed. You must install peer dependencies yourself.
npm WARN @tensorflow-models/facemesh@0.0.3 requires a peer of @tensorflow/tfjs-converter@^1.6.1 but none is installed. You must install peer dependencies yourself.
npm WARN @tensorflow-models/facemesh@0.0.3 requires a peer of @tensorflow/tfjs-core@^1.6.1 but none is installed. You must install peer dependencies yourself.

Reply:
Yeah the version of facemesh we have install isn't quite compatible with the TFJS version we are using (it works just fine, just the dependencies complain). Next update to facemesh should update to the latest TFJS anyways though.